fixtures/futures/tests/bindings/test_futures.py

```diff
@@ -14,7 +14,8 @@ async def test():
             t1 = now()
 
             t_delta = (t1 - t0).total_seconds()
-            self.assertTrue(t_delta < 0.1)
+            # XXX - this is suspect - nothing guarantees this!
+            self.assertLess(t_delta, 0.1)
```

Comment:
Yeah I agree, let's kill the assertions that seem like they could cause intermittent failures again